tests/Spec/Nri/Ui/ClickableText.elm

```diff
@@ -1,72 +1,411 @@
-module Spec.Nri.Ui.ClickableText exposing (spec)
+module Spec.Nri.Ui.ClickableText exposing (..)
```

Comment:
Is there a reason to expose everything instead of the top level `spec`?

Reply:
that was unintended